src/main/java/se/sundsvall/smloader/api/JobsResource.java

```diff
@@ -0,0 +1,70 @@
+package se.sundsvall.smloader.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.constraints.NotNull;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import org.zalando.problem.violations.ConstraintViolationProblem;
+import se.sundsvall.smloader.service.AsyncExecutorService;
+
+import java.time.LocalDateTime;
+
+import static org.springframework.http.MediaType.ALL_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.noContent;
+
+@RestController
+@Validated
+@Tag(name = "Jobs", description = "Jobs resource")
+@RequestMapping("/jobs")
```

Comment:
MunicipalityID should probably be included in this path?

Reply:
This api is only internal for triggering jobs. MunicipalityID is found in CaseMetaDataEnity to control the flow.